notebooks/nessie-iceberg-demo-nba.ipynb

```diff
@@ -0,0 +1,629 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Nessie Spark SQL Demo with NBA Dataset\n",
+    "============================\n",
+    "This demo showcases how to use Nessie Python API along with Spark3 from Iceberg\n",
+    "\n",
+    "Initialize Pyspark\n",
+    "----------------------------------------------\n",
+    "To get started, we will first have to do a few setup steps that give us everything we need\n",
+    "to get started with Nessie. In case you're interested in the detailed setup steps for Spark, you can check out the [docs](https://projectnessie.org/tools/spark/)\n",
+    "\n",
+    "The Binder server has downloaded spark and some data for us as well as started a Nessie server in the background. All we have to do is start Spark\n",
+    "\n",
+    "The below cell starts a local Spark session with parameters needed to configure Nessie. Each config option is followed by a comment explaining its purpose.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import findspark\n",
+    "from pyspark.sql import *\n",
+    "from pyspark import SparkConf\n",
+    "findspark.init()\n",
```

Comment:
the demo fails for me on this step with
```
IndexError                                Traceback (most recent call last)
/srv/conda/envs/notebook/lib/python3.7/site-packages/findspark.py in init(spark_home, python_path, edit_rc, edit_profile)
    142     try:
--> 143         py4j = glob(os.path.join(spark_python, "lib", "py4j-*.zip"))[0]
    144     except IndexError:

IndexError: list index out of range

During handling of the above exception, another exception occurred:

Exception                                 Traceback (most recent call last)
<ipython-input-2-9521cc520592> in <module>
      3 from pyspark.sql import *
      4 from pyspark import SparkConf
----> 5 findspark.init()
      6 
      7 conf = SparkConf()

/srv/conda/envs/notebook/lib/python3.7/site-packages/findspark.py in init(spark_home, python_path, edit_rc, edit_profile)
    144     except IndexError:
    145         raise Exception(
--> 146             "Unable to find py4j, your SPARK_HOME may not be configured correctly"
    147         )
    148     sys.path[:0] = [spark_python, py4j]

Exception: Unable to find py4j, your SPARK_HOME may not be configured correctly
```

Am I missing something?

Reply:
this is due to the failure w/ the `os.link` that you mentioend above. somehow I didn't catch that in testing. Fixed now.